src/gateio_new_coins_announcements_bot/main.py

```diff
@@ -80,288 +80,330 @@ def buy():
+                elif (
+                    float(last_price) < stored_price + (stored_price * coin_sl / 100)
+                    or float(last_price) > stored_price + (stored_price * coin_tp / 100)
+                    and not globals.enable_tsl
+                ):
                     try:
-                        fees = float(order[coin]['_fee'])
+                        fees = float(order[coin]["_fee"])
                         sell_volume_adjusted = float(volume) - fees
 
                         logger.info(
-                            f'starting sell place_order with :{symbol} | {globals.pairing} | {volume} | {sell_volume_adjusted} | {fees} | {float(sell_volume_adjusted)*float(last_price)} | side=sell | last={last_price}',
-                            extra={'TELEGRAM':  'SELL_START'})
+                            f"starting sell place_order with :{symbol} | {globals.pairing} | {volume} | "
+                            + f"{sell_volume_adjusted} | {fees} | {float(sell_volume_adjusted)*float(last_price)} | "
+                            + f"side=sell | last={last_price}",
+                            extra={"TELEGRAM": "SELL_START"},
+                        )
 
                         # sell for real if test mode is set to false
                         if not globals.test_mode:
-                            sell = place_order(symbol, globals.pairing, float(sell_volume_adjusted)*float(last_price), 'sell', last_price)
+                            sell = place_order(
+                                symbol,
+                                globals.pairing,
+                                float(sell_volume_adjusted) * float(last_price),
+                                "sell",
+                                last_price,
+                            )
                             logger.info("Finish sell place_order")
 
-
-                            #check for completed sell order
-                            if sell._status != 'closed':
+                            # check for completed sell order
+                            if sell._status != "closed":
 
                                 # change order to sell remaining
                                 if float(sell._left) > 0 and float(sell._amount) > float(sell._left):
                                     # adjust down order _amount and _fee
-                                    order[coin]['_amount'] = sell._left
-                                    order[coin]['_fee'] = f'{fees - (float(sell._fee) / float(sell._price))}'
+                                    order[coin]["_amount"] = sell._left
+                                    order[coin]["_fee"] = f"{fees - (float(sell._fee) / float(sell._price))}"
 
                                     # add sell order sold.json (handled better in session.json now)
+
+                                    # TODO: check if this makes sense for id
+                                    id = 0
```

Comment:
This was a error flake8 threw at me.
I'm not sure if it makes sense to set id=0 here but it was uninitialized and would not have worked anyway.